features/popularmovies/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'com.android.library'
+apply from: "$project.rootDir/tools/gradle/script/quality.gradle"
+apply from: "$project.rootDir/tools/gradle/base.gradle"
+
+android {
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies{
+    api project(":datasource:local")
+    api project(":datasource:remote")
+    api project(":models:localmodel")
+    api project(":models:remotemodel")
```

Comment:
`:models:remotemodel`, `:models:localmodel` and `:models:datasourcemodel` are exposed as APIs in `: models:mapper `. No need to add them here